libs/@guardian/libs/src/consent-management-platform/sourcepoint.ts

```diff
@@ -80,6 +130,10 @@ export const init = (framework: ConsentFramework, pubData = {}): void => {
 		window.guardian?.config?.tests?.useSourcepointPropertyIdVariant ===
 		'variant';
 
+	const isCorpABTest = isInConsentOrPayABTest();
+
+	console.log('participations', isCorpABTest);
```

Comment:
Is this console.log still required? And if it is, maybe we can use the @guardian/libs logger https://github.com/guardian/csnx/blob/3a93d78e7b599acd58ad01002f8de72b3b58e2c1/libs/%40guardian/libs/src/logger/logger.ts#L88 to only log if the user is subscribed to these logs?